scripts/cell-archive-minio.sh

```
@@ -0,0 +1,171 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+readonly MINIO_IMAGE='minio/minio@sha256:14cea493d9a34af32f524e538b8346cf79f3321eff8e708c1e2960462bd8936e'
+readonly MC_IMAGE='minio/mc@sha256:a7fe349ef4bd8521fb8497f55c6042871b2ae640607cf99d9bede5e9bdf11727'
+readonly CELLD_IMAGE="${CELLD_ARCHIVE_IMAGE:-celld-ci}"
+readonly RUN_ID="cell-archive-${RANDOM}-$$"
+readonly NETWORK="${RUN_ID}-network"
+readonly MINIO="${RUN_ID}-minio"
+readonly TEST_ROOT="$(mktemp -d /tmp/celld-archive.XXXXXX)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)